python/sanlock.c

```diff
@@ -91,8 +105,9 @@ __parse_resource(PyObject *obj, struct sanlk_resource **res_ret)
     res->num_disks = num_disks;
 
     for (i = 0; i < num_disks; i++) {
-        char *p = NULL;
+        const char *p = NULL;
```

Comment:
This is not related to the change. If you want improve this do it sedately after we finish with py3 stuff.